Makefile

```diff
@@ -134,6 +135,15 @@ thinkhazard/static/build/%.css: $(LESS_FILES) .build/node_modules.timestamp
 	sed -e 's#{{PYTHONPATH}}#$(shell .build/venv/bin/python -c "import distutils; print(distutils.sysconfig.get_python_lib())")#' \
 		-e 's#{{WSGISCRIPT}}#$(abspath .build/venv/thinkhazard.wsgi)#' $< > $@
 
+.build/pyramid.zip:
+	mkdir -p $(dir $@)
+	wget -O $@ https://github.com/Pylons/pyramid/archive/master.zip
```

Comment:
I looked at the pyramid repo and there is not tag for that development version. I guess we just need to keep an eye out for the next release and update this link and the setup.py to the released artifact.